packages/graphback-cli/src/config/ConfigBuilder.ts

```diff
@@ -0,0 +1,32 @@
+import { join } from 'path';
+import { ProjectConfig } from './ProjectConfig';
+import { readJsonTemplateConfig } from './readJsonTemplateConfig';
+
+
+/**
+ * Abstraction for Graphback configuration
+ */
+export class ConfigBuilder {
+
+    public config: ProjectConfig
+
+    constructor(creator: () => ProjectConfig) {
+        this.config = creator()
+        this.applyAbsolutePaths();
```

Comment:
We should not be calling this here in `init`, this can be done later in `generate`. Moreover `process.cwd()` is different during `init`. This causes init to fail